development/staging.md

```diff
@@ -1,8 +1,8 @@
 # Staging Environment
 
-We have a staging environment running on AWS which is a scaled down replica of FlowForge Cloud.
-
-It uses the domain flowforge.dev
+We have a staging environment running on AWS which is a scaled down replica of
+FlowForge Cloud. Staging URL and sign in details can be found in the 
```

Comment:
```suggestion
FlowForge Cloud, with a separate domain. Staging URL and sign in details can be found in the 
```

Reply:
Sure, will push this change to main in a bit